apps/graphql/src/apps/itinerary/dataloaders/__tests__/Itineraries.test.js

```diff
@@ -75,3 +99,20 @@ it('parses validity parameters corectly', () => {
     pnum: 1,
   });
 });
+
+it('parses new parameters correctly', () => {
+  expect(parseParametersNew(searchParamsNew)).toMatchInlineSnapshot(`
+Object {
+  "adults": 1,
+  "asc": 0,
+  "children": 0,
+  "curr": "EUR",
+  "dateFrom": "01/05/2019",
+  "dateTo": "03/05/2019",
+  "flyFrom": "prague_cz",
+  "infants": 0,
+  "sort": "quality",
+  "to": "STN",
```

Comment:
I am confused because when I checked the Tequila API these parameters looks different.
https://tequila.kiwi.com/portal/resources/search_api

Probably they release the new version: 
>  Please note, that only active parameters have been described in this document. Parameters that are not described are deprecated.

Reply:
Yes, now that I see it it's a little bit weird... maybe they are outdated.